### PR TITLE
webui: correct the last partitioning getter

### PR DESCRIPTION
**Pull request by KKoukiou**
Previously we would read just the last character of the path, which would give incorrect data if more than 9 partitionings would exist.

Now we parse the digits in the end of the path with a regexp.

### CODE REVIEW

File: ui/webui/src/components/AnacondaWizard.jsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const AnacondaWizard = ({ dispatch, storageData, localizationData, onAddE
     const [showPassphraseScreen, setShowPassphraseScreen] = useState(false);
     const [storageScenarioId, setStorageScenarioId] = useState(window.sessionStorage.getItem("storage-scenario-id") || getDefaultScenario().id);
     const lastPartitioning = useMemo(() => {
-        const lastPartitioningKey = Object.keys(storageData.partitioning || {}).find(path => parseInt(path[path.length - 1]) === Object.keys(storageData.partitioning).length);
+        const lastPartitioningKey = Object.keys(storageData.partitioning || {}).find(path => parseInt(path.match(/\d+$/)[0]) === Object.keys(storageData.partitioning).length);
```
Comment (jelly) [inline ui/webui/src/components/AnacondaWizard.jsx:59]: Would be nice to have a test for this, it still feels a bit fragile.

**Review by jelly — APPROVED** [review]

File: ui/webui/src/components/AnacondaWizard.jsx (hunk 1 above)
Comment (KKoukiou) [reply ui/webui/src/components/AnacondaWizard.jsx:59]: Good point - will add one.

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Review by vojtechtrefny — APPROVED** [review]

File: ui/webui/test/check-storage [hunk 2]
```diff
@@ -234,6 +234,53 @@ class TestStorage(anacondalib.VirtInstallMachineCase):
...
+
+        # Read partitioning data before we do anything
+        created_partitioning = s.dbus_get_created_partitioning()
+
+        # Go to Review step
+        i.open()
+        i.next()
+        i.next()
+        i.next()
+        i.next()
```
Comment (M4rtinK) [inline ui/webui/test/check-storage:255]: It seems to me that we might need to have a library function for reaching a given screen in the future, as the screen order ḿight be more dynamic in the future.

**Review by M4rtinK — APPROVED** [review]
Looks good to me as well. :)

**Review by VladimirSlavik — APPROVED** [review]

**Pull request merged**